### ISSUE TITLE
Wrong azimuth angle

### ISSUE THREAD

**Issue by dinhnhat0401** [mourner/suncalc#6]
Sorry but i find that the sun azimuth must plus 180 degree so:
function getAzimuth(H, phi, dec) {
        return atan(sin(H), cos(H) \* sin(phi) - tan(dec) \* cos(phi));
    }

must be:
function getAzimuth(H, phi, dec) {
        return PI + atan(sin(H), cos(H) \* sin(phi) - tan(dec) \* cos(phi));
    }

**Comment by warpling** [comment]
Is there a reason #35 hasn't been merged?

**Comment by mourner** [comment]
Yeah, hesitating to break backwards compatibility, leading to even more confusion :) I'll deal with this eventually, just didn't get my hands on it yet.

**Comment by warpling** [comment]
Makes sense! Crazy how many forks are downstream also…

**Comment by deqoksijnc** [comment]
If breaking backwards compatibility is a concern than it might be good to add a new interface method which would give the correct result and leave the old one for those who were doing their own workarounds.

**Comment by xqjibz** [comment]
I did not consider backwards functionality when I did the PR, that's my bad, I simply updated my library here and ran with it.  @mourner if you have another solution, I'd be happy to work something up, otherwise no harm no foul on the PR.

**Comment by ultrafez** [comment]
According to [semver](http://semver.org/), the way to avoid breaking users existing apps is to release a new version with a major version number change (i.e. release as 2.0.0). This way, the vast majority of users won't pull it into their project automatically.

**Comment by mourner** [comment]
Implemented as a part of [SunCalc v2](https://github.com/mourner/suncalc/releases/tag/v2.0.0).
